core/common/src/main/java/alluxio/metrics/Metric.java

```diff
@@ -245,8 +245,7 @@ public static Metric from(String fullName, double value) {
     for (int i = tagStartIdx; i < pieces.length; i++) {
       String tagStr = pieces[i];
       if (!tagStr.contains(TAG_SEPARATOR)) {
-        LOG.warn("Unknown tag {}, the tag format should be tagname{}tagvalue", pieces[i],
-            TAG_SEPARATOR);
+        // Unknown tag
```

Comment:
Should we log a warning still?

Reply:
There will be excessive warning loggings, as some metrics (e.g. JVM's) do not conform this convention.

Reply:
We currently use this method to serialize metrics to send to the master. Can we only send the metrics which make sense to send (ie. the Alluxio generated ones which we will aggregate at the cluster level)?